Makefile

```diff
@@ -0,0 +1,11 @@
+OPERATOR_SDK_VERSION ?= v0.18.2
+IMAGE_REGISTRY ?= quay.io
+OPERATOR_IMAGE ?= kubevirt-ssp-operator
+IMAGE_TAG ?= latest
+
+operator-sdk:
+	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
+	chmod 0755 operator-sdk
+
+operator-build:
+	operator-sdk build $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
```

Comment:
We should run `go mod tidy` and `go mod vendor` before building so we don't miss dependencies and add the `--go-build-args` to `operator-sdk build` to make sure we are building with vendored dependencies

Reply:
Let's also add an `operator-push` target to make image updates quicker

Reply:
Calls to operator-sdk should be made with `./operator-sdk [args]`, as `operator-sdk [args]` would use the `$PATH` binary and not the downloaded one in the repo directory

Reply:
Added